motionplan/armplanning/plan_manager.go

```diff
@@ -67,22 +67,28 @@ func newPlanManager(logger logging.Logger, request *PlanRequest) (*planManager,
 }
 
 type atomicWaypoint struct {
-	mp         *cBiRRTMotionPlanner // this is unique to each right now because it embeds the checker, which has the start/goal
-	startState *PlanState           // A list of starting states, any of which would be valid to start from
-	goalState  *PlanState           // A list of goal states, any of which would be valid to arrive at
-
-	// If partial plans are requested, we return up to the last explicit waypoint solved.
-	// We want to distinguish between actual user-requested waypoints and automatically-generated intermediate waypoints, and only
-	// consider the former when returning partial plans.
-	origWP int
+	// this is unique to each (Dan: each what?) right now because it embeds the checker, which has
+	// the start/goal
+	motionPlanner *cBiRRTMotionPlanner
+
+	// A list of starting states, any of which would be valid to start from
+	startState *PlanState
```

Comment:
I'm assuming we want to turn `startState` into a single configuration. No more poses where any given configuration that satisfies a pose are "equally valid".